test/functions_test.py

```diff
@@ -153,15 +155,15 @@
 <BLANKLINE>
 Message: Nothing :)
 <BLANKLINE>
->>> test_params = {"method":"v60", "cups":2, "coffee":60, "water":0.5, "coffee_ratio": 3, "water_ratio":50, "message":"", "digits":3, "coffee_unit": "g", "water_unit": "kg", "grind": 700, "temperature":95, "temperature_unit": "C"}
->>> test_params = filter_params(test_params)
->>> check_ratio_limits(test_params)
+>>> input_params = {"method":"v60", "cups":2, "water":0.5, "coffee_ratio": 3, "water_ratio":50, "message":"", "digits":3, "coffee_unit": "g", "water_unit": "kg", "grind": 700, "temperature":95, "temperature_unit": "C"}
+>>> result_params = get_result(input_params)
+>>> check_ratio_limits(result_params)
```

Comment:
I feel something is wrong about this line. The `coffee` is removed which is unexpected. Even if it remains the same it should be `60 g` but when printing we have `0.06 g`.
Also the water `0.5 kg` does not match new water: `0.001 kg`.